python/src/scipp/__init__.py

```diff
@@ -22,8 +22,21 @@ def custom_formatwarning(msg, *args, **kwargs):
         'You are running a "Debug" build of scipp. For optimal performance use a "Release" build.'
     )
 
-from ._scipp.core import *
 from ._scipp import __version__
+# Import classes
+from ._scipp.core import Variable, DataArray, Dataset, GroupByDataArray, \
+                         GroupByDataset, Dim
```

Comment:
Do we need `Dim`?

Reply:
There is à `test_dim.py` file but I could just remove those tests

Reply:
I don't see why not.